### ISSUE TITLE
JAWS does not properly read tablist length

### ISSUE THREAD

**Issue by gdubeIBM** [FreedomScientific/VFO-standards-support#134]
### Summary

> Brief description of the issue. Please list any specific steps.

When child nodes of a tablist get dynamically updated, the JAWS screen reader does not read the correct count in the automated message. 
Example:
1. Go to https://codepen.io/gdubeIBM/pen/YRpYaY
2. Tab to the "Add Item" button
3. Press the Enter or Space key (to add a new box)
4. Tab to the newly added box
5. BUG: JAWS reads: "Tab 4 of 3...". If you continue to add boxes it will continue to read the wrong tablist length ("5 of 3...", "6 of 3...", etc.). 

### Expected result

> What did you expect to happen?

JAWS should read the correct length of a tablist dynamically.

### Actual result

> What actually did happen?

JAWS does not update the child node count of a tablist after the initial render and proceeds to read the incorrect value to the user. 

### Example

> A test case that demonstrates the issue - provide linked test case, CodePen, JSbin, etc...

https://codepen.io/gdubeIBM/pen/YRpYaY

### Additional Information

This issue is also reproducible if you remove or reorder items from a tablist.

#### JAWS version and build number

JAWS Version 2018.1808.10 ILM
 
#### Operating System and version

OS: Windows 10 Enterprise
Version: 1803

#### Browser and version:

Browser: Firefox Quantum
Version: 60.3.0esr (64-bit)

**Comment by gdubeIBM** [comment]
Hi @stevefaulkner, any idea if this is something Freedom Scientific is planning on resolving?

**Comment by scottaohara** [comment]
Just giving this a quick look, and can confirm the issue with JAWS 2019 and Firefox 63.

That said, trying this out on Chrome the added tabs are announced correctly with JAWS.  

Trying out the demo with IE11 and Edge, neither of these pairings announce the number of tabs within the tablist at all.

**Comment by gdubeIBM** [comment]
Thanks @scottaohara! Any idea how to open an official ticket for this issue?

**Comment by lamador** [comment]
The issue was not reproducible using the latest version of JAWS (2020.2004.66) and Firefox (77.0.1 (64-bit)) on Windows 10.  

Additionally, the latest version of IE11 and Edge are buggy using either JAWS or NVDA when testing the provided code and the [APG 1.2 tabs design pattern.](https://w3c.github.io/aria-practices/examples/tabs/tabs-1/tabs.html).
